### PR TITLE
Opgee 265 mr sid in build

### PR DESCRIPTION
**Pull request by tst-eclamar**
Add a script to be run by jenkins to add MrSID to the build.

### CODE REVIEW

File: scripts/get-gee-build-environment.sh [hunk 1]
```diff
@@ -32,14 +32,14 @@ MIN_GCC_VERSION="4.8"
 : ${PACKAGE_BUILD_DIR:=""}
 
 # Whether to delete the package build directory before exiting:
-: ${CLEAN_PACKAGE_BUILD_DIR:="yes"}
+: ${CLEAN_PACKAGE_BUILD_DIR:="no"}
```
Comment (tst-ppenev) [inline scripts/get-gee-build-environment.sh]: Is changing the default values necessary for building MrSID?

File: scripts/get-gee-build-environment.sh [hunk 2]
```diff
@@ -32,14 +32,14 @@ MIN_GCC_VERSION="4.8"
 : ${PACKAGE_BUILD_DIR:=""}
```
Comment (tst-ppenev) [inline scripts/get-gee-build-environment.sh]: This file has been removed from this repository.  This change should not be part of this pull request.

File: scripts/get-MrSID-environment.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        echo "$SELF_NAME Error: Must be root!"
+	echo
+	exit 1
+    fi
+}
+
+check_root
+
+MRSIDTGZ="MrSID_DSDK-9.5.4.4703-rhel6.x86-64.gcc482.tar.gz"
+echo "MRSIDTGZ=$MRSIDTGZ"
```
Comment (tst-ppenev) [inline scripts/get-MrSID-environment.sh]: What is the purpose of this `echo` command? Is it just for debugging?

File: scripts/get-MrSID-environment.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+# This scripts installs packages, downloads and builds software used to
+# download the Open GEE source code and build it.
+#   It runs on all supported platforms.
+#   This script does not share code with other shell scripts so that it can be
+# distributed as a single file that can be used to obtain a build environment
+# and source code for Open GEE.
+
+SELF_NAME=$(basename "$0")
+
+is_string_value_false()
```
Comment (tst-ppenev) [inline scripts/get-MrSID-environment.sh]: These functions look like code may be used elsewhere, and that can be put in a shell library and re-used.

File: scripts/get-MrSID-environment.sh [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    fi
+}
+
+check_root
+
+MRSIDTGZ="MrSID_DSDK-9.5.4.4703-rhel6.x86-64.gcc482.tar.gz"
+echo "MRSIDTGZ=$MRSIDTGZ"
+wget http://bin.lizardtech.com/download/developer/$MRSIDTGZ -O /tmp/$MRSIDTGZ
+tar -xzf /tmp/$MRSIDTGZ -C /opt
+echo /opt/MrSID_DSDK-9.5.4.4703-rhel6.x86-64.gcc482/Raster_DSDK/lib >>/etc/ld.so.conf
```
Comment (tst-ppenev) [inline scripts/get-MrSID-environment.sh]: This will keep adding lines to `/etc/ld.so.conf`, and there's no counterpart that removes them.

I think it's better to specify a directory on the `ldconfig` command line, or, alternatively, add a file to `/etc/ld.so.conf.d/`.

File: scripts/get-gee-build-environment.sh (hunk 1 above)
Comment (tst-basilhuffman) [reply scripts/get-gee-build-environment.sh]: When running the script on centos6, the build died before the MrSID script was even executing. It looked to be dying around the time it was getting to cleanup of the gtest rpm. It wasn't happening in any of the other platforms so I suggested turning that flag off and seeing if it had an effect.

I assume it must have had an effect. Were there any issues vis a vis centos6 initially? Rhel6?

I'm a bit confused as to why -c exists if it defaults to yes also...
